packages/host-portal-app/src/components/remote-view.tsx

```diff
@@ -19,9 +20,31 @@ const views = createContext<
   ]
 >([{}, () => null]);
 
-async function loadRemoteView(baseUrl: string): Promise<ComponentType | void> {
+async function loadRemoteView(
+  baseUrl: string,
+  loadWithIframeFallback?: ManifestCheck,
+): Promise<ComponentType | void> {
   const response = await fetch(`${baseUrl}/package.json`);
   const manifest = (await response.json()) as Manifest;
+  const type = manifest?.modular?.type;
+
+  if (!type || (type !== 'esm-view' && type !== 'app')) {
```

Comment:
This can be simplified to...

```suggestion
  if (type !== 'esm-view' && type !== 'app') {
```